src/problem1.rb

```diff
@@ -0,0 +1,9 @@
+class Problem1
+  def sum(array)
+    unless array.nil?
+      array.sum
```

Comment:
The description of the problem said: `compute the sum of all __even__ numbers in this array`